code/game/objects/items/weapons/kitchen.dm

```diff
@@ -138,6 +138,13 @@
 	icon_state = "render"
 	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/kitchen/knife/shiv
+	name = "glass shiv"
+	desc = "A haphazardly constructed blade, fashioned from a glass shard wrapped in cloth and tied with wire."
```

Comment:
This item currently doesn't have a force value, you may want to give it one,.

Reply:
Intentional, wanted it to inherit the force of a regular knife